seller_data/models.py

```
@@ -11,8 +12,15 @@ def __str__(self): return self.business_name
 
 
 class SellerVerification(VerificationMixin):
-    seller = models.OneToOneField(Seller, on_delete=models.CASCADE)
+    seller = models.OneToOneField(Seller, on_delete=models.CASCADE, db_index=True)
     agent_rera_id = models.CharField(max_length=12, unique=True, editable=False)
     gstin = models.BigIntegerField(validators=[MaxValueValidator(9999999999)], unique=True, editable=False)
 
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+        original = type(self).objects.get(pk=self.pk)
+        if original.gstin != self.gstin: raise ValidationError('GSTIN cannot be changed once set!')
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)